modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -224,39 +224,44 @@ For additional details on all the Query settings in the lower section of the pan
+The index storage mode options are xref:learn:services-and-indexes/indexes/storage-modes.adoc#memory-optimized-index-storage[Memory Optimized Index Storage], and xref:learn:services-and-indexes/indexes/storage-modes.adoc#standard-index-storage[Standard Index Storage].
 
-*Advanced Settings* for indexes are also provided:
+The settings under *Advanced Settings* for indexes are:
 
-* *Indexer Threads*.
-The number of dedicated threads used by the Index Service.
-The number can be increased on multi-core machines.
+* *Indexer Threads*: 
+Sets the number of threads the Index Service uses.
+You can increase the number of threads dedicated to the Index Service on multi-core machines.
 The default is 0.
 
-* *Num Replica*.
-The default number of index replicas to be created by the Index Service whenever `CREATE INDEX` is invoked.
-For further details, refer to xref:learn:services-and-indexes/indexes/index-replication.adoc#index-replication[Index Replication].
+* *Num Replica*:
+Sets the default number of index replicas the Index Service creates for new indexes. Defaults to 0 (no replicas).
+See xref:learn:services-and-indexes/indexes/index-replication.adoc#index-replication[Index Replication].
 
-* *Indexer Rebalance Settings*.
-When the *Optimize Index Placement On Rebalance* checkbox is checked, Couchbase Server redistributes indexes when rebalance occurs, in order to optimize performance.
-If the checkbox is unchecked (which is the default), such redistribution does not occur.
-For further details, refer to xref:learn:clusters-and-availability/rebalance.adoc#rebalancing-the-index-service[Rebalancing the Index Service].
+* *Indexer Rebalance Settings*:
+Clearing **Optimize Index Placement On Rebalance** (the default) has Couchbase Server only redistribute indexes during a rebalance when the nodes containing them are leaving the cluster.
+Selecting it has Couchbase Server redistribute indexes among the Index Service nodes during to optimize performance.
+See xref:learn:clusters-and-availability/rebalance.adoc#index-redistribution[Index Redistribution].
 
-* *Indexer Log Level*.
-Adjust the logging level.
+* *Indexer Log Level*:
+Sets the logging level.
 The options are: `Silent`, `Fatal`, `Error`, `Warn`, `Info`, `Verbose`, `Timing`, `Debug`, and `Trace`.
 The default is `Info`.
 
-* *Bloom Filter*.
-Whether Bloom filters are enabled for memory management.
-When the *Enable Bloom Filter* checkbox is checked, Bloom filters are enabled; otherwise, they are disabled (which is the default).
+* *Bloom Filter*:
+Clearing *Enable Bloom Filter* (the default) turns off Bloom filters for memory management.
+Selecting *Enable Bloom Filter*, enables Bloom filters. 
 See xref:learn:services-and-indexes/indexes/storage-modes.adoc#per-page-bloom-filters[Per Page Bloom Filters].
 
+* *File Transfer Based Rebalance*:
```

Comment:
It needs to be clarified that "File Transfer Based Rebalance" is applicable only for "Standard Global Secondary" storage mode. For "Memory Optimized", this option is not applicable. It would be good to clarify it here as well as any other relevant sections.

Reply:
I updated several areas to cover this.